app-config.yaml

```diff
@@ -139,6 +139,11 @@ techdocs:
 dynatrace:
   baseUrl: https://your.dynatrace.instance.com
 
+# Score-cards sample configuration. TODO: shall we change this to a better location? 
+# E.g. for plugin DEV environment we start http-server.. shal we do the same here?
+scorecards:
+  jsonDataUrl: https://raw.githubusercontent.com/Oriflame/backstage/feature/jvilimek/score-card-plugin/plugins/score-card/sample-data/ #this is being served via http-server
```

Comment:
Would it make sense to move this into an example folder within this plugin, I'm thinking along the lines of how the example `catalog-info.yaml` files are handled

Reply:
Ok, thanks for explaining. Let's see what the maintainers think then

Reply:
maybe I can at least fix the url so it won't target a feature branch, but a master trunk